sdk-ext-lib/src/androidTest/java/cash/z/ecc/sdk/model/WalletAddressesTest.kt

```diff
@@ -19,14 +17,6 @@ class WalletAddressesTest {
         assertFalse(actual.contains(walletAddresses.shieldedSapling.address))
         assertFalse(actual.contains(walletAddresses.transparent.address))
         assertFalse(actual.contains(walletAddresses.unified.address))
-        assertFalse(actual.contains(walletAddresses.viewingKey))
-    }
-
-    @Test
-    @SmallTest
-    fun new() = runTest {
```

Comment:
This test is omitted completely for now. If we validate that the new Synchronizer parameter in WalletAddresses.new() method is the right way to go, then we can come up with a new test or keep it omitted.